README.md

```diff
@@ -183,6 +183,6 @@ you have to keep compatibility with old code.
 Run the following in your command-line:
 
 ```sh
-$ brew tap luizalabs/teresa-cli
+$ brew tap luizalabs/homebrew-teresa-cli
```

Comment:
Are you sure? I thought homebrew solves the `homebrew` prefix.

Reply:
yes, the problem is the version of osx